src/Microsoft.DotNet.Wpf/tests/UnitTests/WindowsBase.Tests/WindowsBase.Tests.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Nullable>enable</Nullable>
+    <Platforms>x64;x86;arm64</Platforms>
+    <RootNamespace />
+    <TargetFramework Condition="!$(TargetFramework.Contains('windows'))">$(TargetFramework)-windows</TargetFramework>
```

Comment:
This looks weird. Where is TargetFramework set and when does it not get set to the `-windows` suffix?

Reply:
It never gets set with `-windows` in WPF. I would think that it should, but I left that as a follow-up. I _have_ to use `-windows` here or `StaFact` won't give me the right assemblies.

Reply:
https://github.com/dotnet/wpf/issues/10053

Reply:
Understood. So why the Contains condition?